docs/project/about.md

```diff
@@ -6,13 +6,27 @@ Pyodide brings the Python runtime to the browser via WebAssembly, along with
 the Python scientific stack including NumPy, Pandas, Matplotlib, parts of
 SciPy, and NetworkX. The [packages
 directory](https://github.com/iodide-project/pyodide/tree/master/packages)
-lists over 35 packages which are currently available.
+lists over 75 packages which are currently available. In addition it's possible
+to install pure Python wheels from PyPi.
 
 Pyodide provides transparent conversion of objects between Javascript and
 Python. When used inside a browser, Python has full access to the Web APIs.
 
-While closely related to the [iodide project](https://iodide.io), a tool for
-*literate scientific computing and communication for the web*, Pyodide goes
-beyond running in a notebook environment. To maximize the flexibility of the
-modern web, **Pyodide** may be used standalone in any context where you want to
-**run Python inside a web browser**.
+## History
+
+Pyodide was created in 2018 by [Michael Droettboom](https://github.com/mdboom)
+at Mozilla as part of the [iodide project](https://iodide.io), a set of experiments around
+scientific computing and communication for the web.
```

Comment:
It might be better to link to the GitHub repository. I don't know how much longer iodide.io will be up but I suspect pyodide and the relevance of this document will outlive it. 😄 

```suggestion
at Mozilla as part of the [iodide project](https://github.com/iodide-project/iodide), a set of experiments around
scientific computing and communication for the web.
```

Reply:
Thanks @wlach !